tools/depends/target/libandroidjni/LIBANDROIDJNI-VERSION

```diff
@@ -1,5 +1,5 @@
 LIBNAME=libandroidjni
-VERSION=cb68ac77303d1edd95e39a6f72655cdb12f76fd2
+VERSION=76e1d4429ea23f8671cada828a29be9c2a7a3210
 BASE_URL=https://github.com/xbmc/libandroidjni/archive
 ARCHIVE=$(VERSION).tar.gz
 SHA512=070e803d8a6fed549fd9f1a92caea300fed1c6c1e3eba168b0b0cdc008e24644e9085c706715853289025e2f55f964c810c46f79fca464c71d3ffe37f7ae366e
```

Comment:
@ksooo you'll need to update the SHA512 as well.

Reply:
yes, meanwhile I did